### ISSUE TITLE
Redesign of summary report

### ISSUE THREAD

**Issue by HDean** [BiologicalRecordsCentre/Pantheon#45]

**Comment by HDean** [comment]
Added mock-up  and notes from Jon Webb for the summary report.

[20160811_CombinedSummaryReportNotes.docx](https://github.com/BiologicalRecordsCentre/Pantheon/files/417702/20160811_CombinedSummaryReportNotes.docx)
[20160811_CombinedSummaryReport.pptx](https://github.com/BiologicalRecordsCentre/Pantheon/files/417703/20160811_CombinedSummaryReport.pptx)

**Comment by johnvanbreda** [comment]
Waiting for revised thoughts on this from Jon.

**Comment by johnvanbreda** [comment]
Specification now:
After clicking the Combined Summary button, you will be presented with a list of all the different tabular outputs available within Pantheon, with a checkbox beside each. You then tick the ones you want to include and press an "Analyse" button. The system then generates a single page with a list of all the report tables you requested.

**Comment by johnvanbreda** [comment]
I've now implemented revised code for the combined summary as suggested. Note that I have not yet enabled all the reports for inclusion in the output, just a handful of Osiris ones so you can test that the principle works OK. If so, then I'll ensure that all the other reports are enabled.

**Comment by HDean** [comment]
Hi @johnvanbreda,
It all looks good to me. We may need to alter the wording a bit to get people to understand what the different reports are (e.g. Osiris:Habitats).
I did find that when I ran the reports, I was getting an error at the top of the page, although everything appeared to be present. It says: Notice: Array to string conversion in extension_misc_extensions::breadcrumb() (line 242 of D:\web sites\drupal7\sites\www.brc.ac.uk.pantheon\modules\iform\client_helpers\prebuilt_forms\extensions\misc_extensions.php).
Suggests an error with the breadcrumb but it appears to be working ok.
